problem_report.py

```diff
@@ -485,6 +479,28 @@ def write(self, file, only_new=False):
                 file.write(base64.b64encode(block))
                 file.write(b"\n")
 
+    def _make_compressed_block(
+        self, v: typing.BinaryIO, bc
+    ) -> typing.BinaryIO:
+        block = b""
+        outblock = bc.compress(v)
+        if outblock:
+            block += base64.b64encode(outblock)
+            block += b"\n "
+        return block
```

Comment:
nitpick: Inverting the if-condition would make the method use the return early pattern that makes the code a bit more readable:
```suggestion
        if not outblock:
            return b""
        block = base64.b64encode(outblock)
        block += b"\n "
        return block
```
I found this blog post explaining the reasoning quite well: https://medium.com/swlh/return-early-pattern-3d18a41bba8